### DISCUSSION TITLE
Gitlab-Runner Operator no available on OKD 4.15.0-0.okd-2024-03-10-010116

### DISCUSSION THREAD

**Discussion by bobby0724** [okd-project/okd#1952] [General]
Hello 

I have installed the latest version of OKD and I am trying to install the gitlab-runner operator but for some reason is no longer available in the operator HUB

anyone have a process to add it ?

**Comment by bo0ts** [comment]
I would avoid the gitlab-runner operator. It has several bugs, the developers are unresponsive and it offers nearly no benefits over the helm chart approach.

But to answer the actual question: the don't list 4.14 - 4.15 as supported versions anymore https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/operators/gitlab-runner-operator/1.25.0/metadata/annotations.yaml#L15

But this used to be case https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/operators/gitlab-runner-operator/1.23.1/certification/metadata/annotations.yaml#L15

**Comment by bobby0724** [comment]
Thanks for the comment but I am looking for gitlab-runner no github

**Reply by bo0ts** [reply]
My bad, that was a typo. My comment is only for gitlab-runner (as you can also see from all the links I've provided).
